extensions/azurecore/src/azureResource/tree/flatAccountTreeNode.ts

```diff
@@ -178,13 +178,25 @@ class FlatAccountTreeNodeLoader {
 			}
 		}, 500);
 		try {
+
+			// Authenticate to tenants to filter out subscriptions that are not accessible.
+
+			let tenants = this._account.properties.tenants;
+			if (tenants.length !== 0) {
```

Comment:
You don't need this check - if there's no items then filter will just return the empty array anyways.